test.properties

```diff
@@ -4020,6 +4020,10 @@
     # Dev Tools
     #
 
+    test.batch.class.names.excludes[dev-tools]=\
+        **/modules/sdk/**/*GreetingBuilderImplTest.java,\
+        **/modules/sdk/**/*GreetingBuilderTest.java
```

Comment:
@kyle-miho if another test class is added inside of a `gradleTest` directory with a different name, we'll run into the same issue. We need to make sure that all `*Test.java` classes inside of a `gradleTest` dir are excluded.